src/plays/product-entry-form/components/UI/ErrorModal.module.css

```diff
@@ -0,0 +1,41 @@
+.modal{
```

Comment:
Can you please use this similar prefix `.product-entry-form` for this styles as well?